spidermon/contrib/scrapy/monitors/monitors.py

```diff
@@ -578,3 +579,31 @@ def get_threshold(self):
 
         expected_item_extracted = math.ceil(previous_count * threshold)
         return expected_item_extracted
+
+
+@monitors.name("Periodic Item Count Increase Monitor")
+class PeriodicItemCountMonitor(BaseStatMonitor):
+    """Check for increase in item count.
+
+    You can configure the threshold for increase using
+    ``SPIDERMON_ITEM_COUNT_INCREASE`` as a project setting or spider attribute.
+    Use int value to check for x new items every check or float value to check
+    in percentage increase of items.
+    """
+
+    stat_name = "item_scraped_count"
+    threshold_setting = "SPIDERMON_ITEM_COUNT_INCREASE"
+    assert_type = ">="
+
+    def get_threshold(self):
+        crawler = self.data.crawler
+        prev_item_scraped_count = self.stats.get("prev_item_scraped_count", 0)
```

Comment:
Most monitor don't assume a default value now and raise and error if used without it being configured. Maybe we should follow this approach here too?

Reply:
Example : https://spidermon.readthedocs.io/en/latest/monitors.html#spidermon.contrib.scrapy.monitors.monitors.ErrorCountMonitor

Reply:
Thanks for the feedback @VMRuiz I've added raise NotConfigured if SPIDERMON_ITEM_COUNT_INCREASE is not configure.